src/DataViews/SamplingReportDataView.cpp

```diff
@@ -61,6 +62,21 @@ const std::vector<DataView::Column>& SamplingReportDataView::GetColumns() {
   return columns;
 }
 
+[[nodiscard]] float SamplingReportDataView::HalfWidthOfSymmetricizedConfidenceInterval(
+    float percentage) const {
+  const float rate = percentage / 100.0f;
+  const auto confidence_interval =
+      app_->GetConfidenceIntervalEstimator()->Estimate(rate, thread_events_count_);
+
+  return std::max(confidence_interval.upper - rate, rate - confidence_interval.lower) * 100.0f;
+}
+
+[[nodiscard]] std::string SamplingReportDataView::BuildSamplingPercentageReport(
```

Comment:
I would name it something simpler, in particular avoiding "report" as `SamplingReport` is the name of a class. Maybe just "BuildInclusiveExclusiveString" or "BuildPercentageString"?